rero_ils/modules/items/api.py

```diff
@@ -177,8 +177,12 @@ def create(cls, data, id_=None, delete_pid=False,
         """Create item record."""
         cls._item_build_org_ref(data)
         data = trim_barcode_for_record(data=data)
+        # Since the barcode is a mandatory field, we set it to current
+        # timestamp if not given
+        data = generate_item_barcode(data=data)
```

Comment:
this does not correspond to the PO specification. I should  contains the PID value. The same logic should be present for the update.

Reply:
On discussion with  the PO and Benoit, the specs of the barcode has changed to:
f-current_timestamp instead of f-pid